generators/helm.go

```diff
@@ -321,11 +337,58 @@ func updateHelmChartDep(chartPath string) (err error) {
 		}
 		absChartPath = asAbs
 	}
+
+	// Parse chart dependency repositories and add them if not present
+	requirementsYamlPath := path.Join(absChartPath, "requirements.yaml")
+	addedDepRepoList := []string{}
+	if _, err := os.Stat(requirementsYamlPath); err == nil {
+		log.Infof("requirements.yaml found at '%s', ensuring repositories exist on helm client", requirementsYamlPath)
+		bytes, err := ioutil.ReadFile(requirementsYamlPath)
+		if err != nil {
+			return err
+		}
+		requirementsYaml := helmRequirements{}
+		if err = yaml.Unmarshal(bytes, &requirementsYaml); err != nil {
+			return err
+		}
+
+		// Add each dependency repo with a temp name
```

Comment:
Why would you use a temp name for each dependency repo as opposed to the repo name, which I assume is already unique?

Reply:
Actually, disregard. I understand now! :)